packages/webapp/pages/squads/new.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ReactElement, useContext, useEffect, useState } from 'react';
+import { NextSeo, NextSeoProps } from 'next-seo';
+import router, { useRouter } from 'next/router';
+import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
+import Unauthorized from '@dailydotdev/shared/src/components/errors/Unauthorized';
+import { SquadDetails } from '@dailydotdev/shared/src/components/squads/Details';
+import AnalyticsContext from '@dailydotdev/shared/src/contexts/AnalyticsContext';
+import { SquadForm, createSquad } from '@dailydotdev/shared/src/graphql/squads';
+import { useBoot } from '@dailydotdev/shared/src/hooks/useBoot';
+import { useToastNotification } from '@dailydotdev/shared/src/hooks/useToastNotification';
+import {
+  AnalyticsEvent,
+  Origin,
+  TargetType,
+} from '@dailydotdev/shared/src/lib/analytics';
+import SourceIcon from '@dailydotdev/shared/src/components/icons/Source';
+import {
+  SquadTitle,
+  SquadSubTitle,
+  ManageSquadPageContainer,
+  ManageSquadPageMain,
+} from '@dailydotdev/shared/src/components/squads/utils';
+import { cloudinary } from '@dailydotdev/shared/src/lib/image';
+import { MangeSquadPageSkeleton } from '@dailydotdev/shared/src/components/squads/MangeSquadPageSkeleton';
+import { getLayout as getMainLayout } from '../../components/layouts/MainLayout';
+import { defaultOpenGraph, defaultSeo } from '../../next-seo';
+
+const DEFAULT_ERROR = "Oops! That didn't seem to work. Let's try again!";
+
+const seo: NextSeoProps = {
+  title: 'Create your Squad',
+  openGraph: { ...defaultOpenGraph },
+  ...defaultSeo,
+};
+
+const NewSquad = (): ReactElement => {
+  const { isReady: isRouteReady, query } = useRouter();
+  const origin = query.origin as Origin;
+  const { user, isAuthReady, isFetched } = useAuthContext();
+  const { trackEvent } = useContext(AnalyticsContext);
+  const { displayToast } = useToastNotification();
+  const { addSquad } = useBoot();
+  const [form] = useState<Partial<SquadForm>>({
+    public: false,
+  });
+
+  useEffect(() => {
+    trackEvent({
+      event_name: AnalyticsEvent.Impression,
+      target_type: TargetType.CreateSquadPopup,
```

Comment:
This should likely change, just flagging to block for now.

Reply:
Discussion here https://dailydotdev.slack.com/archives/C0592NL7YCU/p1688983153989259

Reply:
We are going to use `page view` and remove this impression.